include/alpaka/dev/DevGenericSycl.hpp

```diff
@@ -34,12 +34,12 @@
 #    include <utility>
 #    include <vector>
 
-namespace alpaka::experimental
+namespace alpaka
 {
     template<typename TElem, typename TDim, typename TIdx, typename TDev>
     class BufGenericSycl;
 
-    namespace detail
+    namespace trait::detail
```

Comment:
Same here, `DevGenericSyclImpl` etc. inside this namespace are not `trait::detail`s. Please leave this namespace.